README.md

```diff
@@ -6,9 +6,20 @@
 Please fill out this survey to help us prioritizing framework support:
 [https://goo.gl/forms/FHHbhptcDx8eDNx92](https://goo.gl/forms/FHHbhptcDx8eDNx92)
 
+# Installation
+
+Official NuGet packages can be referenced from [NuGet.org](https://www.nuget.org).
+
+| Package Name            | Purpose          | Download         |
+| ----------------------- | ---------------- | -----------------|
+| `Elastic.Apm`           |  The core of the Agent, Public Agent API, Auto instrumentation for libraries that are part of .NET Standard 2.0  | [![NuGet Release][ElasticApm-image]][ElasticApm-nuget-url]  |
+| `Elastic.Apm.AspNetCore` | ASP.NET Core auto instrumentation | [![NuGet Release][ElasticApmAspNetCore-image]][ElasticApmAspNetCore-nuget-url] |
+| `Elastic.Apm.EntityFrameworkCore` | Entity Framework Core auto instrumentation | [![NuGet Release][Elastic.Apm.EntityFrameworkCore-image]][Elastic.Apm.EntityFrameworkCore-nuget-url] |
+| `Elastic.Apm.NetCoreAll` | References every .NET Core related elastic APM package. It can be used to simply turn on the agent with a single line and activate all auto instrumentation. | [![NuGet Release][Elastic.Apm.NetCoreAll-image]][Elastic.Apm.NetCoreAll-nuget-url] |
+
 ## Documentation
 
-Docs are located [here](docs).
+Docs are located [here](https://www.elastic.co/guide/en/apm/agent/dotnet/current/index.html). That page is generated from the content of the [docs](docs) folder.
```

Comment:
This link that will be stored by tags and branches (if/when we will have any) even though it always points to docs built from master - maybe https://www.elastic.co/guide/en/apm/agent/dotnet/ is better since it will show other versions?

Reply:
Yes, I also think it's be better - fixed.